app/services/bulk_tagging/search.rb

```diff
@@ -27,7 +27,8 @@ def gds_response
         document_type: document_type,
         page: page,
         q: query,
-        fields: [:content_id, :document_type, :title, :base_path]
+        fields: [:content_id, :document_type, :title, :base_path],
+        search_in: 'title,base_path,details.internal_name'
```

Comment:
After a discussion with @tijmenb I've raised https://github.com/alphagov/publishing-api/pull/771 to make this consistent since we're the only ones using this feature for now.